File: two-sum/brute-force.js
const twoSum = (nums, target) => {
    if (nums.length < 2) {
        return null;
    }

    for (let i = 0; i < nums.length - 1; i++) {
        let secondNum = target - nums[ i ];
        for (let j = i + 1; j < nums.length; j++) {
            if (nums[ j ] == secondNum) {
                return [ i, j ];
            }
        }
    }
}

console.log(twoSum([ 2, 7, 11, 15 ], 9)); // [0, 1]
console.log(twoSum([ 3, 2, 4 ], 6)); // [1, 2]
console.log(twoSum([ 3, 3 ], 6)); // [1, 2]

File: 01-two-sum/brute-force.py
def find_two_sum(nums, target):
    if len(nums) < 2:
        return None

    for i in range(len(nums) - 1):
        secondNum = target - nums[i]
        for j in range(i + 1, len(nums)):
            if nums[j] == secondNum:
                return [i, j]

    return None


print(find_two_sum([ 2, 7, 11, 15 ], 9)) # [0, 1]
print(find_two_sum([ 3, 2, 4 ], 6)) # [1, 2]
print(find_two_sum([ 3, 3 ], 6)) # [0, 1]